OpenShift/helm/nexus-repository-manager/README.md

```diff
@@ -0,0 +1,262 @@
+
+```bash
+$ helm install --name my-release -f my-values.yaml stable/sonatype-nexus
+```
+
+### Persistence
+
+By default a PersistentVolumeClaim is created and mounted into the `/nexus-data` directory. In order to disable this functionality
+you can change the `values.yaml` to disable persistence which will use an `emptyDir` instead.
+
+> *"An emptyDir volume is first created when a Pod is assigned to a Node, and exists as long as that Pod is running on that node. When a Pod is removed from a node for any reason, the data in the emptyDir is deleted forever."*
+
+
+You must enable StatefulSet (`statefulset.enabled=true`) for true data persistence. If using Deployment approach, you can not recover data after restart or delete of helm chart. Statefulset will make sure that it picks up the same old volume which was used by the previous life of the nexus pod, helping you recover your data. When enabling statefulset, its required to enable the persistence.
+
+
+### Recommended settings
+
+As a minimum for running in production, the following settings are advised:
+
+```yaml
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: gce
+    kubernetes.io/tls-acme: true
+
+persistence:
+  storageClass: standard
+  storageSize: 1024Gi
+
+resources:
+  requests:
+    cpu: 250m
+    # Based on https://support.sonatype.com/hc/en-us/articles/115006448847#mem
+    # and https://twitter.com/analytically/status/894592422382063616:
+    #   Xms == Xmx
+    #   Xmx <= 4G
+    #   MaxDirectMemory >= 2G
+    #   Xmx + MaxDirectMemory <= RAM * 2/3 (hence the request for 4800Mi)
+    #   MaxRAMFraction=1 is not being set as it would allow the heap
+    #     to use all the available memory.
+    memory: 4800Mi
+```
+
+## After Installing the Chart
+After installing the chart a couple of actions need still to be done in order to use nexus. Please follow the instructions below.
+
+### Nexus Configuration
+The following steps need to be executed in order to use Nexus:
+
+- [Configure Nexus](https://github.com/travelaudience/kubernetes-nexus/blob/master/docs/admin/configuring-nexus.md)
+- [Configure Backups](https://github.com/travelaudience/kubernetes-nexus/blob/master/docs/admin/configuring-nexus.md#configure-backup)
+
+and if GCP IAM authentication is enabled, please also check:
+- [Enable GCP IAM authentication in Nexus ](https://github.com/travelaudience/kubernetes-nexus/blob/master/docs/admin/configuring-nexus-proxy.md#enable-gcp-iam-auth)
+
+### Nexus Usage
+To see how to use Nexus with different tools like Docker, Maven, Python, and so on please check:
+
+- [Nexus Usage](https://github.com/travelaudience/kubernetes-nexus#usage)
+
+### Disaster Recovery
+In a disaster recovery scenario, the latest backup made by the nexus-backup container should be restored. In order to achieve this please follow the procedure described below:
+- [Restore Backups](https://github.com/travelaudience/kubernetes-nexus#restore)
```

Comment:
Are these external docs necessary?  Do we have any more authoritative docs of our own?

Reply:
A lot of this is legacy from `stable` and I haven't vetted them to see how applicable everything is. I'll clean up the readme, even though it may also delete some useful stuff. It can always be recovered.